distribution/engine/THIRD-PARTY/org.graalvm.python.python-resources-23.1.0/NOTICES

```diff
@@ -0,0 +1 @@
+copyright rights in the Software, and any and all patent rights owned or
```

Comment:
This NOTICES file does not make any sense. This is a sentence taken out of context.